detection-rules/impersonation_human_resources.yml

```
@@ -83,8 +83,19 @@ source: |
       and any(ml.nlu_classifier(body.current_thread.text).entities,
               .name == "request"
       )
-      and any(ml.nlu_classifier(body.current_thread.text).entities,
-              .name in ("urgency", "financial")
+      and (
+        any(ml.nlu_classifier(body.current_thread.text).entities,
+            .name in ("urgency", "financial")
+        )
+        or (
+          any(beta.ml_topic(body.current_thread.text).topics,
+              .name == "Professional and Career Development"
+              and .confidence == "high"
+              and any(ml.nlu_classifier(body.current_thread.text).intents,
+                      .name != "benign"
+              )
+          )
+        )
```

no need to do NLU while in the any loop for topics

not sure if this suggestion has the correct intention/grouping logic, it's hard to tell in the GH suggestions
```suggestion
      and (
        any(ml.nlu_classifier(body.current_thread.text).entities,
            .name in ("urgency", "financial")
        )
        or (
          any(beta.ml_topic(body.current_thread.text).topics,
              .name == "Professional and Career Development"
              and .confidence == "high"
          )
          and any(ml.nlu_classifier(body.current_thread.text).intents,
                  .name != "benign"
          )
        )
      )
```